### PR TITLE
Arch 17925/improve cfengine code generation

### PR DESCRIPTION
**Pull request by amousset**

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: rudder-lang/src/generators/cfengine/syntax.rs [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+                        .collect::<Vec<String>>()
+                        .join("\n")
+                ))
+                .collect::<Vec<String>>()
+                .join("\n")
+        )
+    }
+}
+
+pub struct Technique {
```
Comment (gpoblon) [inline rudder-lang/src/generators/cfengine/syntax.rs:304]: don't you want to keep the description?

Comment (amousset) [reply rudder-lang/src/generators/cfengine/syntax.rs:304]: Description is a documentation field for directive creators, I don't think it really belongs in the generated policies.

Comment (gpoblon) [reply rudder-lang/src/generators/cfengine/syntax.rs:304]: ok

**Pull request merged**